examples/sumbetadiff.R

```diff
@@ -1,3 +1,9 @@
-parX <- c(1, 52)
-parY <- c(5.5, 20.5)
-sumBetadiff(parX, parY, 0.9, cutB = 0.8, cutW = 0.2)
+parX <- c(1, 52) # Control group's parameters
+parY <- c(5.5, 20.5) # Treatment group's parameters
+sumBetadiff(
+  parX = parX,
+  parY = parY,
+  coverage = 0.9, # 90 % credible interval
```

Comment:
```suggestion
  coverage = 0.9,
```
this will be clear with the right argument name